apps/system/js/app_text_selection_dialog.js

```diff
@@ -3,10 +3,13 @@
 (function(exports) {
   var DEBUG = false;
   var _id = 0;
-  /**
-   * Text Selection Dialog of the AppWindow
-   */
 
+  // Timer id obtained in _launchCutOrCopiedTimer() shared by all apps. If cut
+  // command or copy command has been performed within CUT_OR_COPIED_TIMEOUT_MS,
+  // it will be non-null.
+  var _cutOrCopiedTimeoutId = null;
```

Comment:
In this file, let create a `AppTextSelectionDialogGlobalStates` constructor, and a variable `globalStates` in this closure to keep the instance.